src/util/util.ts

```diff
@@ -26,6 +26,10 @@ export function isDefined(value: any): boolean {
   return value !== undefined && value !== null;
 }
 
+export function isDomObject(value: any): boolean {
```

Comment:
I think this one is not needed, simple `sth instanceof HTMLElement` works ok even in E11, and is already used in the library for same purpose:
https://github.com/ng-bootstrap/ng-bootstrap/blob/4f61496fff898996e92bd8928d83d5536382aa90/src/datepicker/datepicker-input.ts#L506

Reply:
I removed this method and used instanceof check as suggested